Orange/widgets/model/owneuralnetwork.py

```diff
@@ -0,0 +1,94 @@
+import re
+from AnyQt.QtCore import Qt
+
+from Orange.data import Table
+from Orange.modelling import NNLearner
+from Orange.widgets import gui
+from Orange.widgets.settings import Setting
+from Orange.widgets.utils.owlearnerwidget import OWBaseLearner
+
+
+class OWNNLearner(OWBaseLearner):
+    name = "Neural Network"
+    description = "A multi-layer perceptron (MLP) algorithm with " \
+                  "backpropagation."
+    icon = "icons/NN.svg"
+    priority = 90
+
+    LEARNER = NNLearner
+
+    activation = ["identity", "logistic", "tanh", "relu"]
+    solver = ["lbfgs", "sgd", "adam"]
+
+    learner_name = Setting("Neural Network")
+    hidden_layers_input = Setting("100")
+    activation_index = Setting(3)
+    solver_index = Setting(2)
+    alpha = Setting(0.0001)
+    max_iterations = Setting(200)
+
+    def add_main_layout(self):
+        box = gui.vBox(self.controlArea, "Network")
+        self.hidden_layers_edit = gui.lineEdit(box, self,
+                                               "hidden_layers_input",
+                                               label="Neurons per hidden "
+                                                     "layer:",
```

Comment:
Perhaps layer -> layers? It is not obvious the user can input a list. Not many users will find the tooltip.